airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -385,18 +385,22 @@ def attempt_task_runs(self):
                     )
                     self.pending_tasks.append(ecs_task)
                 else:
-                    self.send_message_to_task_logs(
-                        logging.ERROR,
-                        "ECS task %s has failed a maximum of %s times. Marking as failed. Reasons: %s",
-                        task_key,
+                    reasons_str = ", ".join(failure_reasons)
+                    self.log.error(
+                        "ECS task %s has failed %s times. Marking as failed. Reasons: %s" % task_key,
```

Comment:
```suggestion
                        "ECS task %s has failed %s times. Marking as failed. Reasons: %s",
```

And pass `task_key` as first parameter